client/src/languages/rPython.ts

```diff
@@ -0,0 +1,327 @@
+// 		}
+// 	}
+
+// 	switch(pyNode.kind) {
+// 		case 'code': 
+// 			let importedFrames : { name:string, url:string }[] = [];
+// 			for (var ant of pyNode.antecedents) {
+// 			let imported = <Graph.ExportNode>ant
+// 			// console.log(imported);
+// 			// console.log(imported.value.data);
+// 			importedFrames.push({ name: imported.variableName, url: (<Values.DataFrame>imported.value).url })
+// 			}
+
+// 					console.log(importedFrames);
+// 					let src = pyNode.source
+// 					console.log(src)
+// 					let hash = Md5.hashStr(src)
+// 					let body = {"code": src,
+// 									"hash": hash,
+// 									"frames": importedFrames}
+// 					return await getEval(body);
+
+// 		case 'export':
+// 			let pyExportNode = <Graph.PyExportNode>node
+// 			let exportNodeName= pyExportNode.variableName
+// 			let exportsValue = <Values.ExportsValue>pyExportNode.code.value
+// 			return exportsValue[exportNodeName]
+// 		}
+// },
+// parse: (code:string) => {
+// 	console.log(this.language);
+// 	return new rPythonBlockKind(code, this.language);
+// },
+// bind: (scopeDictionary: Langs.ScopeDictionary, block: Langs.Block) => {
+// 		let pyBlock = <rPythonBlockKind>block
+// 		// return getCodeExports(scopeDictionary, pyBlock.source)
+// 		let dependencies:Graph.PyExportNode[] = [];
+// 		let node:Graph.PyCodeNode = {
+// 			language:"python", 
+// 			antecedents:[],
+// 			exportedVariables:[],
+// 			kind: 'code',
+// 			value: undefined,
+// 			source: pyBlock.source
+// 		}
+		
+// 		let url = this.serviceURI.concat("/exports")
+// 		let hash = Md5.hashStr(pyBlock.source)
+		
+// 		// let hash = "Md5.hashStr(pyBlock.source)"
+// 		let body = {"code": pyBlock.source,
+// 								"hash": hash,
+// 								"frames": Object.keys(scopeDictionary)
+// 							}
+// 		let headers = {'Content-Type': 'application/json'}
+// 		async function getExports() {
+// 			try {
+// 				const response = await axios.post(url, body, {headers: headers});
+// 				// console.log(response.data.exports)
+// 				// console.log(response.data.imports)
+// 				for (var n = 0 ; n < response.data.exports.length; n++) {
+// 					// console.log(response.data.exports[n]);
+// 					let exportNode:Graph.PyExportNode = {
+// 						variableName: response.data.exports[n],
+// 						value: undefined,
+// 						language:"python",
+// 						code: node, 
+// 						kind: 'export',
+// 						antecedents:[node]
+// 						};
+// 					dependencies.push(exportNode) 
+// 					node.exportedVariables.push(exportNode.variableName)
+// 					console.log(node);
+// 				}
+// 				for (var n = 0 ; n < response.data.imports.length; n++) {
+// 					if (response.data.imports[n] in scopeDictionary) {
+// 						let antecedentNode = scopeDictionary[response.data.imports[n]]
+// 						node.antecedents.push(antecedentNode);
+// 					}
+// 				}
+// 				return {code: node, exports: dependencies};
+// 			} catch (error) {
+// 				console.error(error);
+// 			}
+// 		}
+// 		return getExports()
+// 	}
+// }
```

Comment:
Can you delete all the commented-out code? If we ever need it, we have it somewhere in the Git history!